src/pages/CarsList/CarsList.scss

```diff
@@ -13,12 +15,28 @@
     color: blueviolet;
   }
 }
-
 .mb-2.btn-group {
-  display: block;
+  display: flex;
   margin-top: 2%;
   margin-left: 2%;
+  padding: 0.25rem;
+  width: 40%;
+  background-color: $lightGrey;
   button:focus {
-    background-color: black;
+    background-color: $black;
+    color: $white;
+    box-shadow: none;
   }
```

Comment:
make this a generic class, and use the react state to show which one is selected